api/resource/resource.go

```diff
@@ -47,6 +48,8 @@ var BuildAnnotations = []string{
 	kioutil.LegacyPathAnnotation,
 	kioutil.LegacyIndexAnnotation,
 	kioutil.LegacyIdAnnotation,
+
+	konfig.HelmGeneratedAnnotation,
```

Comment:
Review note: Adding this here causes the Annotation to not get exposed to the user.